### DISCUSSION TITLE
Welcome to ServiceNet Discussions!

### DISCUSSION THREAD

**Discussion by johnhbenetech** [benetech/ServiceNet#1475] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by greggish** [comment]
Hi there, Greg here, I lead the @openreferral initiative, which developed [the HSDS and api protocols](http://docs.openreferral.org). I help communities figure out the resource directory interoperability and sustainability problem, and I think ServiceNet presents a very promising opportunity for the field. Looking forward to see where this goes!
